tasks/download-atom-shell-task.coffee

```diff
@@ -25,7 +25,6 @@ module.exports = (grunt) ->
 
   getArch = ->
     switch process.platform
-      when 'win32' then 'ia32'
```

Comment:
The default should probably still be 32-bit, but having an explicit override should definitely be A Thing